ui/lib/utils/DI.md

```diff
@@ -0,0 +1,52 @@
+# Dependency Injection (DI) Documentation
```

Comment:
You may like to move it to our Wiki :)  That's where our development docs locates.
https://github.com/pingcap/tidb-dashboard/wiki

Reply:
Ok, it's just a demo-like implementation now. I'll reorganize it and refine docs after `debug the api`.